### PR TITLE
Recipe for diffpy.pdfmorph (tests may be wrong)

### PR DESCRIPTION
**Pull request by RomanMatthew**
Checklist

- [y] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [y] Source is from official source
- [y] Package does not vend other packages
- [y] Build number is 0
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by RomanMatthew** [comment]
@CJ-Wright This is the pull request I made for pdfmorph - something went really wrong!

File: recipes/pdf_recipe/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run_constrained:
+    - python 3.*
```
Comment (CJ-Wright) [inline recipes/pdf_recipe/meta.yaml]: I would put this as python >=3 for both host and run.

File: recipes/pdf_recipe/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
```
Comment (CJ-Wright) [inline recipes/pdf_recipe/meta.yaml]: This needs runtime requirements

Comment (RomanMatthew) [reply recipes/pdf_recipe/meta.yaml]: I thought that's what run_constrained was?

Comment (CJ-Wright) [reply recipes/pdf_recipe/meta.yaml]: https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

Comment (RomanMatthew) [reply recipes/pdf_recipe/meta.yaml]: I didn't see any mention of runtime requirements in that link - does it have to do with the entry points? Sorry for all the confusion about this

File: recipes/pdf_recipe/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3
+    - pip
+  run_constrained:
```
Comment (CJ-Wright) [inline recipes/pdf_recipe/meta.yaml]:
```suggestion
  run:
    - matplotlib
    - diffpy.utils
    - scipy
    - numpy
```

File: recipes/pdf_recipe/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,53 @@
+{% set name = "diffpy.pdfmorph" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/diffpy/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 35a9c5803b0bb3e4f4487544efe44d4063c38cc3601c33c893182359db4ece8d
```
Comment (CJ-Wright) [inline recipes/pdf_recipe/meta.yaml]:
```suggestion
  sha256: 699952c747048db4a383e81fa867887a86480200a56d7a51bdd1ad43d25b392d
```

File: recipes/pdf_recipe/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+  # doc_url: doc_url to be released in a later version.
+  dev_url: https://github.com/diffpy/diffpy.pdfmorph
+
+extra:
+  recipe-maintainers:
+    - chiahaoliu
+    - CJ-Wright
+    - pavoljuhas
+    - sbillinge
+
```
Comment (CJ-Wright) [inline recipes/pdf_recipe/meta.yaml]:
```suggestion
```

**Comment by CJ-Wright** [comment]
It looks like this can't be noarch, can you remove the `noarch: python` put in a `skip: True  # [py<3]` and remove the `>=3` from the python requirements?

**Pull request closed without merging**